app/controllers/projects_controller.rb

```diff
@@ -6,12 +6,16 @@ class ProjectsController < ApplicationController
   load_resource except: %i[show index]
 
   def index
-    @projects = Project.default_scoped
     if params[:query].presence
-      @projects = @projects.search(params[:query])
+      @projects = Project.includes(:user)
+                         .search(params[:query])
+                         .sort_by do |p|
```

Comment:
Fix indent, change, use reorder(:created_at)
Also this order should be enforced in else branch without code duplication.

Reply:
done1